app/models/interest.rb

```diff
@@ -83,9 +83,9 @@ def summary
       target_type.underscore.to_sym.l + ": " +
       (target ? target.unique_format_name : "--")
   end
-  alias_method :text_name, :summary
+  alias text_name summary
```

Comment:
The difference is lexical scope.  See https://github.com/bbatsov/ruby-style-guide#alias-method-lexically for: discussion, including why alias is preferred here (and I suppose why it's the RuboCop default), and when `alias_method` is preferred.  (RuboCop tries to catch those cases.)